app/routes/new-signup.js

```diff
@@ -30,7 +32,15 @@ export default class NewSignupRoute extends Route {
           '',
           toastNotificationTimeoutOptions
         );
-        setTimeout(() => window.open(GOTO_URL, '_self'), 2000);
+        if (this.featureFlag.isDevMode) {
+          setTimeout(
+            () =>
+              window.open('http://localhost:5500/goto.html?dev=true', '_self'),
```

Comment:
Same here, local host?

Reply:
What is being done here

Reply:
added feature flag..will get remove in future